src/test/javascript/admin.controller.test.js

```diff
@@ -153,63 +207,30 @@ describe("AdminController", function () {
                 "sn": "tst01name"
             };
         });
-        it("should set scope.emptySelect to true when scope.selectedGroupingsPaths is empty", function () {
+
+        it("should set scope.emptySelect to true when scope.selectedGroupingsPaths is empty", () => {
             scope.removeFromGroupsCallbackOnSuccess(res);
             expect(scope.emptySelect).toBeTrue();
         });
     });
 
-    describe("removeFromGroups", function () {
```

Comment:
It is very difficult to see changes when portions of code is moved around like this. In the future, maybe separate writing the code and sorting the parts into separate pull requests. It will allow easier reading.